### ISSUE TITLE
Linking with skiplist does not appear to be needed

### ISSUE THREAD

**Issue by hselasky** [silentbicycle/hopscotch#4]
```
diff --git a/Makefile b/Makefile
index bb7eea8..a80b9b3 100644
--- a/Makefile
+++ b/Makefile
@@ -21,7 +21,7 @@ CSTD +=               -std=c99 #-D_POSIX_C_SOURCE=200112L -D_C99_SOURCE
 CFLAGS +=      ${CSTD} -g ${WARN} ${CDEFS} ${CINCS} ${OPTIMIZE}
 
 LDFLAGS +=
-MAIN_LDFLAGS += ${LDFLAGS} -lskiplist -L${BUILD} -lhopscotch
+MAIN_LDFLAGS += ${LDFLAGS} -L${BUILD} -lhopscotch
 
 TEST_CFLAGS =  ${CFLAGS}
 TEST_LDFLAGS =  ${LDFLAGS} -ltheft

```

**Comment by hselasky** [comment]
Running your tool on a large dataset revealed some issues. See the attached patch for fixes.
[hpscotch.txt](https://github.com/silentbicycle/hopscotch/files/3527411/hpscotch.txt)

**Comment by silentbicycle** [comment]
You're absolutely right that it doesn't need the skiplist library. That was a dependency early on and never got removed. Good catch!

I'm looking at your other patch now. Both will go on the develop branch and in the next release. Thanks.

**Comment by silentbicycle** [comment]
These are fixed in the 0.1.2 release. Thanks.

**Comment by hselasky** [comment]
Thank you! I'll give it a spin too when I get back to my project :-)
